### DISCUSSION TITLE
Do I need a full zip file? 我需要完整的zip文件吗？

### DISCUSSION THREAD

**Discussion by YanYiGe2023** [ThioJoe/Windows-Super-God-Mode#30] [Q&A]
I just need Super_God_Mode.ps1 and SuperGodMode-EasyLauncher.bat 
Or do you need to download the whole zip file of the version? 
Because I noticed that there are also powershell script files in the Extra Tools folder, can the program still work without them？
 
我只需要Super_God_Mode.ps1和SuperGodMode-EasyLauncher.bat 
还是需要下载版本的整个zip文件？ 
因为我注意到Extra Tools文件夹中也有powershell脚本文件，缺失它们程序还可以工作吗？

**Comment by ThioJoe** [comment]
Only the .ps1 file is truly necessary, because that is the actual program. The .bat file is just for convenience to make the .ps1 file easier to run, but the .ps1 file can be run manually with the instructions in the ReadMe - it's just an extra command in PowerShell.

So no, the whole zip file is not necessary. I just added "extra tools" and other things in case anyone might find them useful or interesting.
